pkg/grpc/api.go

```diff
@@ -10,7 +10,6 @@ import (
 	"errors"
 	"fmt"
 	"strconv"
-	"time"
```

Comment:
Please update the protos as well and generated files in this PR.

Reply:
updated the protos and added files in this PR.